projects/cli-files/README.md

```diff
@@ -10,7 +10,6 @@ Timebox: 4 days
 
 ## Objectives:
 
-- Install and use [cobra][cobra]
 - Use go build/go install/go get etc
```

Comment:
- Make a go module
- Describe the basics of a process and process lifecycle
- Accept arguments on the CLI

Reply:
Done!